give/src/components/Image/Image.js

```diff
@@ -0,0 +1,64 @@
+// Based on v3
+// https://www.aemcomponents.dev/content/core-components-examples/library/core-content/image.html
+
+import "./Image.css";
+import lava1 from "../../assets/lava1.jpeg";
+
+const LinkWrapper = ({ href, children }) => {
+  return href ? (
+    <a className="cmp-image__link" href={href}>
+      {children}
+    </a>
+  ) : (
+    children
+  );
+};
+
+export const Image = ({
+  src = "", // string
+  alt = "", // string
+  caption = "", // string
+  href = "", // string - URL
+  popupTitle = true, // boolean
+  classes = "", // string
+}) => {
+  if (src === "" && alt === "") return null;
+
+  return (
+    <div className={`image cru-image${classes ? " " + classes : ""}`}>
+      <div className="cmp-image">
+        <LinkWrapper href={href}>
+          <img
+            src={src}
```

Comment:
Something that is nice with material ui is if an image is broken on the Avatar component it actually handles showing an icon so there isn't the broken image default by the browser

Reply:
That's helpful. I haven't begun to think about adding MUI into the mix of this just yet. I've just been converting the AEM components to React. I'll edit the PR's description to include a link to the AEM documentation I'm following.